pages/instrumenting.md

```diff
@@ -125,12 +125,28 @@ information for the request.
 * Flags - Provides the ability to create and communicate feature flags. This is how
 we can tell downstream services that this is a "debug" request.
 
-Check [here](https://github.com/openzipkin/brave/blob/e474ed1e1cd291c7ebc6830c58fdba0a6318fdd2/brave-http/src/main/java/com/github/kristofa/brave/http/BraveHttpHeaders.java) for the format
+Check [here](https://github.com/openzipkin/brave/blob/e474ed1e1cd291c7ebc6830c58fdba0a6318fdd2/brave-http/src/main/java/com/github/kristofa/brave/http/BraveHttpHeaders.java) for the format.
 
 Finagle provides mechanisms for passing this information with HTTP and Thrift
 requests. Other protocols will need to be augmented with the information for
 tracing to be effective.
 
+**Sampling decisions are made at the edge of the system**
```

Comment:
Not sure if we should clarify this as instrumentation sampling? Ex this is true for instrumentation.. but there may be storage tier sampling which might need it's own section. Storage tier sampling acts like instrumentation whereas a trace is thrown out or retained. However, it is controlled independently (ex a server flag, or an adaptive rate stored in zookeeper)